CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # \[Unreleased\]
 
+- Nix: adds a new dev shell that contains only Holochain binaries and Rust, named `holochainBinaries`.
```

Comment:
:+1: 

i was tempted to put some nix related changes here previously as well, however the release cycles don't apply to nix code (yet)